MyAmazingRPG/MyAmazingRPG/src/Character.java

```diff
@@ -0,0 +1,44 @@
+import static java.lang.System.*;
+
+public class Character {
+
+    static int characterCount;
+
+    public Character() {
+        characterCount++;
+    }
+
+    String name;
+    int health;
+    int strength;
+    int defense;
+    int damage;
+    boolean isAlive;
+
+    public void attack(Character other) {
+        damage = strength - other.defense;
+        if (damage < 0) {
+            damage = 0;
+        }
```

Comment:
You already have the reference to `other` here, so if you want other to `.takeDamage(int damage)` -> this can be here in place, right?